packages/asc-ui/src/components/Footer/FooterLinkList/FooterLinkListItem.tsx

```diff
@@ -16,6 +17,7 @@ const FooterListItem = styled(ListItem)`
 
   ${LinkStyle} {
     color: ${themeColor('tint', 'level1')};
+    background-color: 'transparent';
```

Comment:
Plz change to `transparent` without the quotes (btw stupid that stylelint didn't spot this...)